### ISSUE TITLE
Evaluation accuracy with frozen graph lower than value given Deeplabv3+ Xception 65 baackbone

### ISSUE THREAD

**Issue by Selventhiranraj** [tensorflow/models#10242]
When using the [xception65_cityscapes_trainfine](http://download.tensorflow.org/models/deeplabv3_cityscapes_train_2018_02_06.tar.gz) **frozen graph** alone instead of using the entire checkpoint provided to calculate the validation accuracy on the Cityscapes dataset with Eval scales of 1, I receive a total mIOU score of 63%, which is quite a bit lower than the stated mIOU score of 78.79%. I have used the same way of preprocessing and calculating accuracy as the eval.py script. Is this difference in accuracy known, and is there a difference between the **frozen graph** given and the model that scored 78.79%.
